edgeless_orc/src/lib.rs

```diff
@@ -32,6 +32,10 @@ pub async fn edgeless_orc_main(settings: EdgelessOrcSettings) {
     let (mut orchestrator, orchestrator_task) = orchestrator::Orchestrator::new(settings.clone()).await;
 
     let orchestrator_server = edgeless_api::grpc_impl::orc::OrchestratorAPIServer::run(orchestrator.get_api_client(), settings.orchestrator_url);
+    let orchestrator_coap_server = edgeless_api::coap_impl::orchestration::CoapOrchestrationServer::run(
+        orchestrator.get_api_client().node_registration_api(),
+        std::net::SocketAddrV4::new("0.0.0.0".parse().unwrap(), 7051),
```

Comment:
Is there any reason why the binding address and port number are hard-coded rather being configurable in `orchestrator.toml` like for gRPC?

Reply:
No, needs to be configurable.